docusaurus.config.js

```diff
@@ -153,6 +153,7 @@ module.exports = {
         {
           title: "Community",
           items: [
+            /* What is the best way to handle the Slack reference below?*/
```

Comment:
@akeller Mind having a look at this? It was the only reference I was iffy on removing.

Reply:
A good ol' fashion delete. I got it 😄 commit incoming.

Reply:
I remove the whole <a></a>.